packages/blocks/src/api/raw-handling/phrasing-content-reducer.js

```diff
@@ -22,7 +22,7 @@ export default function( node, doc ) {
 			wrap( doc.createElement( 'em' ), node );
 		}
 
-		if ( textDecorationLine === 'line-through' ) {
+		if ( textDecoration === 'line-through' ) {
```

Comment:
Why not check against both? What if this introduces a regression with other tools/browsers?

Reply:
I guess if we check a more general property we should check if the string contains `line-through`. In any case, if `textDecorationLine` doesn't work in some browsers, we should leave a comment here.

Reply:
Updated this to check both attributes, check `textDecoration` using `includes()`, and added a comment.